test/utils/deploy/deploy.go

```diff
@@ -199,3 +199,25 @@ func isMachineConfigPoolsUpdatedAfterDeletion(nro *nropv1alpha1.NUMAResourcesOpe
 
 	return true, nil
 }
+
+func DeployNROScheduler() (*nropv1alpha1.NUMAResourcesScheduler, error) {
+
+	nroSchedObj := objects.TestNROScheduler()
+	if err := e2eclient.Client.Create(context.TODO(), nroSchedObj); err != nil {
+		return nil, err
+	}
+	if err := e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj); err != nil {
```

Comment:
note this is not waiting for the scheduler to be actually available

Reply:
done